addons/mail/static/src/components/composer/composer.xml

```diff
@@ -48,12 +48,27 @@
                         </t>
                     </div>
                 </t>
+                <t t-if="composer.thread and composer.thread.mass_mailing">
+                    <div class="o_Composer_subject">
+                        <input class="o_Composer_subjectInput" type="text" placeholder="Subject"
+                               t-esc="composer.subjectInputContent"
+                               t-on-focusin="_onFocusinSubject"
+                               t-on-focusout="_onFocusoutSubject"
+                               t-on-keyup="_onKeyupSubject"
+                               t-on-input="_onInputSubject"
+                               t-ref="subject"/>
```

Comment:
should inline attributes on standard nodes (and the multiline syntax would be to use standard tab not visual align, and close on new line)